pandas/tests/reshape/merge/test_join.py

```diff
@@ -831,3 +840,44 @@ def test_join_multiindex_one_level(join_type):
         index=pd.MultiIndex.from_tuples([(2, 1)], names=["b", "a"]),
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize(
```

Comment:
can you also add a non-categorical on either side here (should be the same result)

Reply:
i know we probably have that but this is a nice regression test

Reply:
We are already joining over 2 columns, where one is not categorical?
Or do you mean another test where ``second`` is not categorical?